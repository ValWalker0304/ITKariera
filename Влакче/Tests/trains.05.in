Add 68193 Zimnitsa P 3
Add 55720 Plovdiv P 9
Add 43512 GOryahovitsa F 20
Add 23396 Montana F 7
Add 91312 Varna F 19
Add 25069 Karnobat F 3
Add 87133 Varna P 10
Add 66460 Shumen P 6
Next
Add 15389 Blagoevgrad P 5
Add 30492 Mezdra P 6
Add 88691 Plovdiv F 14
Add 48700 GOryahovitsa P 7
Add 96969 Tulovo F 14
Add 97938 Yambol P 5
Travel
Add 73777 Kulata F 30
Add 96907 Mezdra F 34
Add 72986 Yambol P 3
Add 71527 Sindel P 6
Next
Travel
Travel
Add 62275 Zimnitsa P 10
Travel
Travel
Travel
End