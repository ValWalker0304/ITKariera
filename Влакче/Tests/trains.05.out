66460 Shumen P 6
97938 Yambol P 5
96907 Mezdra F 34
96907 Mezdra F 34
73777 Kulata F 30
62275 Zimnitsa P 10
71527 Sindel P 6
72986 Yambol P 3
